runTavernTestsWithSimulator.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
```

Comment:
This file I think needs a MIT license header.

Reply:
Thanks, added.